libs/blocks/caas-config/caas-config.js

```diff
@@ -363,6 +363,7 @@ const UiPanel = () => html`
   <${Input} label="Use Light Text" prop="useLightText" type="checkbox" />
   <${Input} label="Use Overlay Links" prop="useOverlayLinks" type="checkbox" />
   <${Input} label="Show total card count at top" prop="showTotalResults" type="checkbox" />
+  <${Input} label="Hide date for on-demand content" prop="hideDateInterval" type="checkbox" />
```

Comment:
L3 seems to fail the linter `/* global ClipboardItem */` - might have been a linting update since this file had been last touched

Reply:
(nitpick) I would label this checkbox as "Hide Event date",
Since it can be used to hide the event date range, not only on-demand :)

Reply:
@cmiqueo the requestors asked for the events to still show the dates for upcoming/live - so for clarity I would leave the title as is